benchmarks/ps-experiment/README.md

```diff
@@ -0,0 +1,176 @@
+
+## Manifest Preparation
+
+We already generated the experiment Yaml, If you like to generate from scratch, follow the steps below.
+
+> Note:
+>
+> 1. The scripts has been verified to be working on VKE, If you need to run on other clouds, you still need to make some changes.
+> 2. Due to testing environments difference, default yaml use `aibrix-container-registry-cn-beijing.cr.volces.com`,
+you can remove this prefix to directly fetch images from dockerhub.
+
+
+| Experiment Name           | Gateway     | Routing Algorithm   | vLLM Version            | KV Cache                    | Explanation                                                                             |
+|---------------------------|-------------|---------------------|-------------------------|-----------------------------|-----------------------------------------------------------------------------------------|
+| aibrix_kvcache_0.6.1.yaml | AIBrix      | Kubernetes          | 0.6.1-aibrix-optimized  | AIBrix client + distributed | AIBrix deployment with KV cache enabled; used to test distributed KV cache behavior.    |
+| aibrix_naive.yaml         | AIBrix      | Kubernetes          | 0.7.0-upstream          | w/o                         | AIBrix baseline setup without optimizations; used as a naive comparison point.          |
+| k8s_stack.yaml            | K8S Service | Kubernetes          | 0.7.0-upstream          | w/o                         | General Kubernetes stack configuration for running inference workloads. K8s Service     |
+| ps_stack.yaml             | PS Router   | Session cache aware | 0.7.0-upstream          | w/o                         | Helm-rendered YAML for the PS (Parameter Server) setup; includes Router, not real naive |
+| ps_k8s_stack.yaml         | PS Router   | Session cache aware | 0.7.0-lmcache-optimized | LM Cache Client lib         | PS full Setup with session aware router and LMCache.                                    |
+
+
+### AIBrix control plane
+
+Please follow the [installation guidance](https://aibrix.readthedocs.io/latest/getting_started/installation/installation.html) to install AIBrix.
+
+### Generate All YAML Files from Scratch
+
+- Create raw yaml files
+   - PS yaml is generated from helm chart.
+   - AIBrix yaml is from official documentation
+- Modify YAMLs Based on Environment Needs
+ - Node Affinity: Adjust for clusters that have other GPUs (e.g., test environments).
+ - Model Persistence & Downloading: Update volume settings and model download steps (e.g., for TOS & VKE usage).
+ - Parameters: Tune engine-related settings and resources such as `--max-model-len`, `VLLM_RPC_TIMEOUT`, `memory limits`, etc.
+- Ensure Completeness: Each YAML file should include all necessary components for deployment.
+
+Feel free to use text diff to compare the differences.
+
+> note: raw yaml source
+
+Scripts to render PS yamls
+```
+helm template vllm vllm/vllm-stack -f stack.yaml > ps_stack.yaml
```

Comment:
Add a comment, where to get stack.yaml and naive.yaml.

Reply:
Also, if naive.yaml is used from production stack repo to generate ps_k8s_stack.yaml then it is missing VLLM_RPC_TIMEOUT env variable.

Reply:
their generated one doesn't have it, but we can add it for fair comparison

Reply:
Done, add file source